f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -187,14 +191,71 @@ def _create_payload(self, loadbalancer, service):
     @serialized('LoadBalancerManager.create')
     @log_helpers.log_method_call
     def create(self, loadbalancer, service, **kwargs):
-        # TODO(qzhao): Future work
-        pass
+        self._pre_create(service)
+        super(LoadBalancerManager, self).create(
```

Comment:
It may have deadlock issue, but we should solve the problem in another commit.

Reply:
Yes， it has.